### ISSUE TITLE
Watcher cannot be paused on nodes

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#197]
Originally reported of Google Code with ID 141.

```
The `gnt-cluster watcher pause` command only works on the master node (as expected), but then it also just sets the pause file on the master node. The result is that yes, down instances are not restarted anymore, but the node daemons are restarted on nodes.

Either the watcher pause file should be replicated to all nodes, or the watcher pause command should be runnable on any node.
```
Originally added on 2011-02-14 08:53:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
As this is in the `gnt-cluster` command I expect a `gnt-cluster watcher pause` to pause watcher on every node. How about optional arguments to specify nodes/groups on which you want to pause the watcher for certain period of time?
```
Originally added on 2011-03-08 09:32:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released

**Comment by Ganeti-Issues-Migrator** [comment]
```
Undo wrong status update to 'released'.
```
Originally added on 2012-10-22 09:09:12 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
Assigning to Michael as he recently worked on this. I believe it's on track to be fixed in 2.7.
```
Originally added on 2012-12-20 12:24:38 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Yes, I believe this is fixed in the following commit which will be in 2.7:

commit 7c4bd15
Author: Michael Hanselmann <hansmi@google.com>
Date:   Tue Dec 11 16:48:03 2012 +0100

    Set watcher pause on all nodes
    
    Instead of just setting the watcher pause file, which isn't replicated,
    RPC is used to set it on all nodes (where possible). This means that
    after an outage of the master node and a following master-failover, the
    watcher will still be paused.
    
    The RPC is attempted on all nodes. Errors for nodes marked offline are
    ignored. All other errors are returned in the form of an exception
    (there is no other option given the LUXI API).
    
    Signed-off-by: Michael Hanselmann <hansmi@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2012-12-20 12:44:43 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
